src/java.desktop/windows/classes/sun/print/PrintServiceLookupProvider.java

```diff
@@ -92,24 +84,20 @@ public PrintServiceLookupProvider() {
             win32PrintLUS = this;
 
             // start the local printer listener thread
-            AccessController.doPrivileged((PrivilegedAction<Thread>) () -> {
```

Comment:
Can you remove the `@SuppressWarnings("removal")` from this method after this change?

Reply:
ok